src/components/cc-domain-management/cc-domain-management.js

```diff
@@ -103,7 +103,7 @@ export class CcDomainManagement extends LitElement {
     this._sortedDomains = null;
 
     new LostFocusController(this, '.delete-domain', ({ suggestedElement }) => {
-      if (suggestedElement != null) {
+      if (suggestedElement != null && suggestedElement instanceof HTMLElement) {
```

Comment:
thinking: when I see that, maybe the suggested element should always be a focusable element because this the whole point of this controller.

Reply:
Yeah I think you're right :thinking: I won't handle it in this PR but let's create an issue + handle it in a separate one?

Reply:
Created the #1161 issue to track this, not sure when we'll tackle this (doesn't seem that long to do but I didn't want to postpone this PR).
Thanks for this really good idea :wink: